packages/dts-generator/src/phases/dts-code-gen.ts

```diff
@@ -255,10 +255,10 @@ function genModule(ast: Module) {
   text += `declare module "${ast.name}" {` + NL;
   text += APPEND_ITEMS(ast.imports, genImport);
   text += APPEND_ITEMS(ast.exports, genExport);
-  (text += APPEND_ITEMS(ast.namespaces, (namespace: Namespace) =>
+  ((text += APPEND_ITEMS(ast.namespaces, (namespace: Namespace) =>
     genNamespace(namespace, { export: namespace.export }),
   )),
```

Comment:
This looks strange. Shouldn't the comma at the end of line 260 rather be a semicolon? I see no reason why this should be a sequence...

Reply:
doesn't change the behavior and was like this before, but you are right. I'll fix it to make the code more readable.